docs/experimental-machine-learning/problem-child.md

```diff
@@ -61,4 +61,4 @@ You can optionally choose to refresh your index mapping from within Kibana:
 
 #### 4. Verify enrichment fields
 
-Any documents corresponding to Windows process events should now be enriched with `problemchild.*`
\ No newline at end of file
+Any documents corresponding to Windows process events should now be enriched with `problemchild.*`. If you have the blocklist for the model enabled, you might also see the field `blocklist_label` appear in certain documents.
```

Comment:
Do you think you should also expand on what enabling the block list means/looks like, or do you think it is obvious enough from the blogs

Reply:
I can change the wording a bit to make it more obvious.